mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -1128,27 +1146,15 @@ def add_metric(
                     if mep and value is not None and not math.isnan(value):
                         setattr(mep.status, metric, value)
 
-        add_metric(
-            "error_count",
-            "count(error_count)",
-            error_count_res,
-        )
-
-        add_metric(
-            "avg_latency",
-            "avg(latency)",
-            avg_latency_res,
-        )
-        add_metric(
-            "result_status",
-            "max(result_status)",
-            drift_status_res,
-        )
-
+        for metric_name, result in metric_name_to_result.items():
+            add_metric(
+                metric_name,
+                metric_name_to_function_and_column_name[metric_name][1],
+                result,
+            )
         self._enrich_mep_with_last_request(
```

Comment:
unlike `tdengine`, here we always enrich the mep with the `last_request` value, even if that metric wasn't part of the provided `metrics`. I guess it's fine though we could be more consistent by adding an `if` clause to ensure the user actually requested the `last_request` metric before including it.